cql3/expr/expression.hh

```diff
@@ -218,10 +218,15 @@ const column_value& get_subscripted_column(const subscript&);
 /// Only columns can be subscripted in CQL, so we can expect that the subscripted expression is a column_value.
 const column_value& get_subscripted_column(const expression&);
 
+/// Represents token(c1, c2) function on LHS of an operator relation.
+/// args contains arguments to the token function.
+struct token {
+    std::vector<expression> args;
```

Comment:
Need updates to anything that treats nested expressions, like fill_prepare_context or search_and_replace.

Reply:
Damn I forgot about it, done.